extractor/filesystem/language/haskell/cabal/testdata/valid

```diff
@@ -0,0 +1,5 @@
+constraints: any.AC-Angle ==1.0,
+             any.ALUT ==2.4.0.3,
```

Comment:
Maybe also add non-version lines like the ones from https://github.com/awakesecurity/spectacle/blob/92b08102979c1d9709cf2483cae1ddb25301d4e4/cabal.project.freeze#L74 and verify that they're ignored